src/_help/doc-beta.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Beta
+---
+
+- TOC
+{:toc}
+
+Use the `x-beta` property inside an Operation, a schema or a parameter object to identify it as beta.
+The `x-beta` property is a boolean.
+
+Add or remove this property is a strucutal change.
+A change on a beta component, can't be breaking.
+
+## Example usage
+
+The following screen capture shows the rendering of adding two code samples `cURL` and `Ruby` to your Operation.
+
```

Comment:
This is a bad copy-pasta. Can you remove the line?

```suggestion
```

Reply:
Indeed. Sorry